oap-server/server-starter/src/main/resources/meter-analyzer-config/go-runtime.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+expSuffix: instance(['service'], ['instance'], Layer.GENERAL)
+metricPrefix: meter
+metricsRules:
+  - name: instance_golang_heap
+    exp: instance_golang_heap
+  - name: instance_golang_stack
+    exp: instance_golang_stack
+  - name: instance_golang_gc_time
+    exp: instance_golang_gc_time.downsampling(SUM)
```

Comment:
I had tested, and the new instance_golang_gc_time  meter in UI represents the sum of all gc puase time that collected in a one minute. For example, if the meter interval is 15s, then go2sky will collect gc_time four times in a minute, and UI will show the sum of four collected gc_pause_time, but gc will happened more than 4 time in one minute(around 500 times one minute),
so these sum  of incomplete gc pause time may confuse users?

Reply:
What do you mean? I am not following. The sum of time cost of GC should be reasonable, no matter how many time you collected and reported.

Reply:
I mean the ui only shows the sum of time cost of **collected** GC (only four times in one minute if the interval is 15s) not the **all** GC(around 500 times in one minute) in one minute, so if ui shows the gc_cost_time is 0.5ms at 20:01, actually, the total gc_pause_time would more than 0.5ms at 20:01.  I just confuse about that, thanks for helping me.

Reply:
Which type actually this metric is? Is it a counter or gauge? If it is a gauge, then it should always report value from last time, the report period should not matter.

Reply:
The current collection method is a gauge, Because it just collect the latest single gc's pause time, I think I should change the collection method to collect the total gc_pause_time in the collect period, thanks.

Reply:
Yes, so, you could check whether there is a hook/callback to measure this. Reading GC pause time periodly seems a back practice.

Reply:
I use stats.PauseTotal to collect the total gc time which represents the sum of gc time after the instance started. So it is a counter meter. And I use increase('PT1M') to calculate the sum of one minute. So the metrics is instance_golang_gc_time.increase('PT1M'). I have tested that and it worked, is this reasonable?

Reply:
Yes, then this is the GC pause time per minute metric.

Also, please try to re-word all the metrics names, they should be more accurate, formal and clear.

Reply:
> Yes, then this is the GC pause time per minute metric.
> 
> Also, please try to re-word all the metrics names, they should be more accurate, formal and clear.

Ok, thanks, I will do this work.